src/style.css

```diff
@@ -1,3 +1,71 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+}
+
+ul,
+li,
+nav {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__link {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__link {
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  height: 60px;
+  line-height: 60px;
+  display: block;
+  margin-left: 20px;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.nav__link:hover {
+  color: var(--blue-color);
+  text-decoration: none;
+}
+
+li:first-child a {
+  margin-left: 0;
+}
+
+.is-active {
+  color: var(--blue-color);
+}
+
+.is-active::after {
+  display: block;
+  content: '';
+  background-color: var(--blue-color);
+  height: 4px;
+  position: relative;
```

Comment:
The `position: relative;` style for the `.is-active::after` pseudo-element might not be necessary unless you plan to position it relative to its normal position. If not needed, consider removing it to simplify the code.